clients/src/main/java/org/apache/kafka/common/metrics/stats/Count.java

```diff
@@ -16,30 +16,17 @@
  */
 package org.apache.kafka.common.metrics.stats;
 
-import java.util.List;
-
 import org.apache.kafka.common.metrics.MetricConfig;
 
 /**
  * A {@link SampledStat} that maintains a simple count of what it has seen.
+ * This is a special kind of {@link Sum} that always records a value of {@code 1} instead of the provided value.
+ *
+ * See also {@link TotalCount} for a non-sampled version of this metric.
  */
-public class Count extends SampledStat {
-
-    public Count() {
-        super(0);
-    }
-
+public class Count extends Sum {
```

Comment:
I'm not generally a fan of building tall class hierarchies, but in this case, it seemed like the cleanest way to delegate to `Sum`, which has the advantage of code de-duplication, but more importantly makes the code more self-documenting... I.e., this metric is exactly the same as Sum, except that it always increments by `1`.